documentation/CAMARA-Security-Interoperability.md

```diff
@@ -66,6 +66,8 @@ All network connections MUST use TLS 1.2 or better.
 
 The OIDC Authorization Code Flow is defined in [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html)
 
+It is RECOMMENDED that signed authentication requests be used, as specified by [OIDC](https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests). The same key MAY be used for signing the authentication request as is used for [client authentication](#client-authentication). This document recommends in the [client credentials section](#client-authentication) that the `aud` value SHOULD be the URL of the Authorization Server's [Token Endpoint](https://openid.net/specs/openid-connect-core-1_0.html#TokenEndpoint). The same recommendation is given for the `aud` value of the signed request object.
```

Comment:
Both in [OIDC](https://openid.net/specs/openid-connect-core-1_0.html#RequestObject) and [RFC 9101](https://www.rfc-editor.org/rfc/rfc9101.html#section-4), the valid value for the `aud` field should be the issuer of the authorization server. I don't see any specification that recommends the Token Endpoint for signed request object.

Even in the [CIBA](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.7.1.1) specification, it is also established that the valid value should be the issuer of the authorization server.

Reply:
@garciasolero
The CAMARA specification [specifically](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md#client-authentication) makes this recommendation.

However, this applies to client authentication, and for authorisation code flow, it is only the `/token` endpoint where the client is authenticated. For signing the authorisation request, I agree that is does not necessarily follow that the `aud` should also be the URL of the token endpoint.

Reply:
Yes, you are right, hence my comment. The recommendations apply depending on what type of assertion it is, and we should be aligned with what the standards say.

Reply:
[OIDC Core](https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests) refers to [\[RFC9101\]](https://www.rfc-editor.org/rfc/rfc9101.html#name-explicit-endpoints), under section [Explicit Endpoints](https://www.rfc-editor.org/rfc/rfc9101.html#name-explicit-endpoints), it is RECOMMENDED good practice to explicitly state the intended interaction endpoints, in this context the Authorization endpoint (authorization_endpoint)

Reply:
```suggestion
It is RECOMMENDED that signed authentication requests be used, as specified by [OIDC](https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests). The same key MAY be used for signing the authentication request as is used for [client authentication](#client-authentication). 

This document recommends the value of `aud` (Audience) claim MUST be the Authorization endpoint as RECOMMENDED by [\[RFC9101\]](https://www.rfc-editor.org/rfc/rfc9101.html#name-explicit-endpoints).
```